src/components/titles/index.js

```diff
@@ -275,6 +330,23 @@ function draw(gd, titleClass, options) {
                 this.text(d || ' ')
                     .call(svgTextUtils.positionText, attributes.x, attributes.y);
             });
+
+        if(subtitleEnabled) {
+            if(!subtitleTxt) setPlaceholder(subtitleEl, subtitlePlaceholder);
+            else subtitleEl.on('.opacity', null);
+            subtitleEl.call(svgTextUtils.makeEditable, {gd: gd})
+                .on('edit', function(text) {
+                    Registry.call('_guiRelayout', gd, 'title.subtitle.text', text);
+                })
+                .on('cancel', function() {
+                    this.text(this.attr('data-unformatted'))
+                        .call(titleLayout);
+                })
+                .on('input', function(d) {
+                    this.text(d || ' ')
+                        .call(svgTextUtils.positionText, subtitleEl.attr('x'), subtitleEl.attr('y'));
+                });
```

Comment:
Nicely added editable subtitles!
Please consider adding a jasmine test similar to https://github.com/plotly/plotly.js/blob/7ffcdd7d602d52e8a0e75d766ae18fe8e9eb54fb/test/jasmine/tests/titles_test.js#L1390-L1413
for this feature.

Thank you!

Reply:
Also adding a `config_test` above here would be nice: 
https://github.com/plotly/plotly.js/blob/7ffcdd7d602d52e8a0e75d766ae18fe8e9eb54fb/test/jasmine/tests/config_test.js#L382-L386

Reply:
And perhaps one test here:
https://github.com/plotly/plotly.js/blob/7ffcdd7d602d52e8a0e75d766ae18fe8e9eb54fb/test/jasmine/tests/plot_api_react_test.js#L419

Reply:
Done.